src/napari/_qt/qt_event_loop.py

```diff
@@ -209,7 +227,7 @@ def get_qapp(
         app.installEventFilter(QtToolTipEventFilter())
 
     if app.windowIcon().isNull():
-        app.setWindowIcon(QIcon(kwargs.get('icon')))
+        app.setWindowIcon(_svg_path_to_icon(kwargs.get('icon')))
```

Comment:
Function _svg_path_to_icon expected an existing path, not none. So maybe we should use the `[]` operator to get an exception in a proper place? 
```suggestion
        app.setWindowIcon(_svg_path_to_icon(kwargs['icon']))
```